src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellItemRenderer.cs

```diff
@@ -85,7 +86,21 @@ public override AView OnCreateView(LayoutInflater inflater, ViewGroup container,
 
 			HookEvents(ShellItem);
 			SetupMenu();
-
+			var appearance = ShellContext.Shell.GetAppearanceForPivot(ShellItem);
+			if (_bottomView.Background is ColorDrawable background)
```

Comment:
This initializes it correctly but If the `TabBarBackgroundColor` property changes at runtime, the color will not be updated, right?

Reply:
@jsuarezruiz  I verified that the TabBarBackgroundColor is updated correctly at runtime.